plugins/ci/pkg/ci/auto_detection_test.go

```diff
@@ -0,0 +1,45 @@
+package ci
+
+import (
+	"testing"
+
+	"github.com/fairwindsops/insights-plugins/ci/pkg/models"
+	"github.com/stretchr/testify/assert"
+)
+
+// RunCommand runs a command and prints errors to Stderr
+func TestAutoDetection(t *testing.T) {
+
+	cfg, err := ConfigFileAutoDetection("./testdata/repo1")
+	assert.NoError(t, err)
+
+	expected := models.Configuration{
+		Manifests: models.ManifestConfig{
+			YamlPaths: []string{
+				"failing.yaml",
+				"passing.yaml",
+				"yml/failing_2.yml",
+				"yml/json/failing_3.json",
+			},
+			Helm: []models.HelmConfig{
+				{
+					Name:       "helm-release-pruner-json",
```

Comment:
Should the name be `{Chart.yaml#name}` (as is) or `{path}/{Chart.yaml#name}`. Because charts w/ the same `Chart.yaml#name` in different folders would have their names clashing

Reply:
I think it's safe to assume that `Chart.yaml#name` is unique. If it's not, let's print a warning